Makefile

```diff
@@ -26,8 +27,13 @@ faktory:
 faktory/kill:
 	docker stop faktory
 
-README.md: README.tpl src/lib.rs
-	cargo readme > README.md
+.PHONY: faktory/tls
+faktory/tls:
+	docker compose -f docker/compose.yml up -d --build
```

Comment:
`--build` flag needed otherwise a cached image will be used, even if we update the NGINX or Faktory images versions in their corresponding dockerfiles. I am also adding this flag on the CI, though of course no images are cached since we are starting a-fresh each time. But still.